cep-cache-output.md

```diff
@@ -22,6 +22,19 @@ There are many downsides to the behavior of `conda-build`: it's very implicit, h
 
 For the v1 spec we are attempting to formalize the workings of the "top-level" build. For this, we introduce a new top-level key `cache`, that has the same values as a regular output.
 
+## Motivating Cases
+
+This section describes specific example cases which would be used by this workflow.
+
+### Dividing a single package variant into modular chunks
+
+Some packages have a monolithic build system but have optional components at runtime. The most common example of this case is splitting a package into development, runtime libraries, and runtime tools; the `dav1d-feedstock` is an example of this. `dav1d-dev` contains the developer artifacts: package config files and headers. `dav1d` contains the command line utility of the same name. `libdav1d7` contains the shared libraries that other packages link to dynamically at runtime. The artifacts in these packages cannot be built separately or doing so would result in redundant work, so the package is built all together, installed to the PREFIX, then split into the output packages.
+
+### Reusing cached build artifacts to build multiple variants
+
+I've heard that instead of building package from scratch for each python version, some maintainers try to rebuild multiple variants of a package from a cached working directory. This might be useful if upstream hasn't designed their build system to build C and Python APIs separately.
```

Comment:
This is the case which I feel that you may be writing for. It's a bit nebulous to me, and I don't do it, so I'm uncertain about how to best support it (if at all).